tests/integration/cartography/intel/gcp/test_compute.py

```diff
@@ -118,14 +128,64 @@ def test_transform_and_load_gcp_instances_and_nics(neo4j_session):
          'project-abc',
          'projects/project-abc/zones/europe-west2-b/instances/instance-1-test/networkinterfaces/nic0',
          '10.0.0.3',
+         None,
          TEST_UPDATE_TAG),
         (instance_id2,
          'europe-west2-b',
          'project-abc',
          'projects/project-abc/zones/europe-west2-b/instances/instance-1/networkinterfaces/nic0',
          '10.0.0.2',
+         'test',
          TEST_UPDATE_TAG)
     ])
+    print(actual_nodes)
+    print(expected_nodes)
```

Comment:
Are these needed? If the assertion fails it should print out where the differences exist, right?